feat(instances): Fix error message in scw-fetch-ssh-keys in windows i… (#4376)
scaleway/docs-content: pages/instances/troubleshooting/fix-error-message-in-scw-fetch-ssh-keys.mdx

---
meta:
  title: Fix error when fetching SSH keys on Windows Server 2022

>   description: This page helps you fix the error message when fetching ssh keys on Windows Server 2022

Reviewer: ```suggestion
  description: This page guides you through resolving the error that occurs when fetching SSH keys on Windows Server 2022.
```

content:
  h1: Fix error when fetching SSH keys on Windows Server 2022

>   paragraph: This page helps you fix the error message when fetching ssh keys on Windows Server 2022

Reviewer: ```suggestion
  paragraph: This page guides you through resolving the error that occurs when fetching SSH keys on Windows Server 2022.
```

tags: windows server 2022 ssh-key
dates:
  validation: 2025-02-06
  posted: 2023-02-06
categories:
  - compute
---

After enabling the OpenSSH server on Windows Server 2022 and Windows Server 2022 Core instances you will

> need to fetch the SSH keys of your project using `Scw-Fetch-SSH-Keys`. When running this command for the
> first time, you will see the following error message displayed :

Reviewer: ```suggestion
need to fetch the SSH keys of your project using `Scw-Fetch-SSH-Keys`. When you run this command for the first time, the following error message will be displayed:
```

```
PS C:\Users\Administrator> Scw-Fetch-SSH-Keys
Remove-Item : Cannot find path 'C:\ProgramData\ssh\administrators_authorized_keys' because it does not exist.
At C:\Program Files\WindowsPowerShell\Modules\ScalewayEcosystem\0.0.10.3\ScalewayEcosystem.psm1:54 char:2
+     Remove-Item $authorized_keys
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\ProgramData\...authorized_keys:String) [Remove-Item], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand

processed file: c:\ProgramData\ssh\administrators_authorized_keys
Successfully processed 1 files; Failed processing 0 files
```

> The ssh keys will still be fetched, but this message may appear again if the file

Reviewer: ```suggestion
The SSH keys will still be fetched, but this message may reappear in the file.
```.

C:\ProgramData\ssh\administrators_authorized_keys if deleted.

> To avoid this message, simply update the `ScalewayEcosystem` package using the folllowing

Reviewer: ```suggestion
To avoid this message, simply update the `ScalewayEcosystem` package using the folllowing
```

> commands :

Reviewer: ```suggestion
commands:
```
